READING_LISTS/ReproducibilityAndOpenScience.md

```diff
@@ -0,0 +1,42 @@
+This list is adapted from Lucia Magis Weinberg's [list of resources for getting started with open science and reproducibility](https://luciamagisweinberg.wordpress.com/2017/03/23/reproducibilityopen-science-resources/)
+
+### Some overviews and timelines of the "Reproducibility Crisis"
+
+Wagenmakers, E.-J. (2015). [A perfect storm: The record of a revolution](http://www.in-mind.org/article/a-perfect-storm-the-record-of-a-revolution). In-Mind Magazine.
+
+[Replicability and Reproducibility Debate](http://www.bps.org.uk/what-we-do/bps/governance/boards-and-committees/research-board/research-board-resources/replicability-and-reproducibility-debate/replicability-and-reproducibility-debate) organized by the BPS at the Royal Society on 26 May 2016, with presentations from Marcus Munafò, Roger Watt, Dorothy Bishop, Chris Chambers, Kathryn Sharples, Nick Brown and Prateek Buch. Go straight to the youtube video of this debate [here](https://www.youtube.com/watch?v=tTuZ-IEc0Eg&feature=youtu.be)
+
+Bastian, H. (2016). Reproducibility Crisis Timeline: Milestones in Tackling Research Reliability. [[Blog]](http://blogs.plos.org/absolutely-maybe/2016/12/05/reproducibility-crisis-timeline-milestones-in-tackling-research-reliability/) Absolutely Maybe.
```

Comment:
Absolutely maybe is the name of the blog, but I will remove it as it's unnecessary and slightly confusing